sdv/constraints/tabular.py

```diff
@@ -460,6 +468,8 @@ def _fit(self, table_data):
         self._validate_columns_exist(table_data)
         self._is_datetime = self._get_is_datetime(table_data)
         self._dtype = table_data[self._column_name].dtypes
+        if self._is_datetime:
```

Comment:
how does the `self._get_is_datetime` method differ from the `is_datetime_type` function? Can we get away with using only one of them? Also, maybe we should just do the datetime validation here since we are already getting whether or not it is a datetime type

Reply:
`_get_is_datetime` checks both, value and column, to be `datetime`. The reason behind is that we end raising a value error inside that function when one of the values is `datetime` and the other one is not.